[TT-4744] (#1892)
TykTechnologies/tyk-docs: tyk-docs/content/tyk-on-prem/installation/kubernetes/helm-chart.md

> ## Other Tyk Components

Reviewer: ```suggestion
## Other Tyk components
```

> ### Installing Tyk Self-managed Control Plane

Reviewer: ```suggestion
### Installing MDCB, the Tyk Self-managed control plane
```

> If you are deploying the Master Data Centre in an *MDCB* deployment then you can set the `mdcb.enabled` option in your `values.yaml` to true to add the MDCB component to your cluster.

Reviewer: ```suggestion
Deploying the MDCB ( Master Data Centre Bridge) component enables multi-cluster, multi data-centre API management from a single Dashboard.

To deploy MDCB in your Kubernetes cluster, set `mdcb.enabled` in your `values.yaml`  to `true`.

Check [Tyk Control plane](https://tyk.io/docs/tyk-multi-data-centre/) for more configuration details.
```

> This enables multi-cluster, multi Data-Centre API management from a single Dashboard.

Reviewer: ```suggestion
```

Check [Tyk Gateway Sharding]({{< ref "/content/advanced-configuration/manage-multiple-environments/manage-multiple-environments.md#api-sharding" >}}) for more details.

> ## Other Tyk components

Reviewer: ```suggestion
## Other Tyk Components
```

### Installing Tyk Enterprise Developer Portal
If you are deploying the **Tyk Enterprise Developer Portal**, set the appropriate values under the `enterprisePortal` section in your `values.yaml`. Please visit [Tyk Enterprise Developer Portal installation]({{< ref "/content/tyk-stack/tyk-developer-portal/enterprise-developer-portal/install-tyk-enterprise-portal.md#launch-the-tyk-enterprise-developer-portal-using-helm" >}}) for a step by step guide.

> ### Installing Tyk Self-managed control plane (MDCB)

Reviewer: ```suggestion
### Installing Tyk Self-managed Control Plane (MDCB)
```
